opba-facade-protocol-api-shared/pom.xml

```diff
@@ -22,5 +22,27 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>de.adorsys.opba</groupId>
```

Comment:
We should not have a dependency on API here, a facade should forward the entity with the mapper. So here you may have duplicate entity but it needs to be decoupled from API

Reply:
done